core/include/cubos/core/tel/metrics.hpp

```diff
@@ -96,7 +96,7 @@ namespace cubos::core
 /// @note The macro uses `__FUNCTION__` to register the metric name.
 
 #ifdef CUBOS_PROFILING
-#define CUBOS_PROFILE() ::cubos::core::ScopeProfiler _profilingTimeTracker(__FUNCTION__, __FILE__, __LINE__)
+#define CUBOS_PROFILE() ::cubos::core::tel::ScopeProfiler _profilingTimeTracker(__FUNCTION__, __FILE__, __LINE__)
```

Comment:
This is kind of useless now, no? Considering we now have spans
Should we remove it? We should only add the `begin` and `end` metrics to spans if `CUBOS_PROFILING` is enabled.